flex_model/core/hook_function.py

```diff
@@ -185,6 +185,9 @@ def __init__(
         self.module_name = module_name
         self.expected_shape = expected_shape
 
+        # TODO (mchoi): If editing function not passed (ie. just doing
```

Comment:
If we have todos - where should we track these tasks. Github issues? Clickup? Or try and link the two?

Reply:
This one should already be on clickup. Gonna look into adding an issue tracker for git, which can probably be linked to clickup